src/main/java/calculator/StringAdder.java

```diff
@@ -0,0 +1,52 @@
+package calculator;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringAdder {
+    public static final int ZERO = 0;
+    public static String regex = ",|:";
+
+    public static int splitAndSum(String text) {
```

Comment:
```suggestion
    public static int sum(split 된 변수) {
```

메소드마다 하나의 책임을 가질수있도록 분리하면 좋을듯해요!